docs/output.md

```diff
@@ -12,7 +12,8 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
 
 Quality check:
 
-- [SeqKit](#seqkit) for input amino acid sequences quality check (QC)
+- [SeqFu](#seqfu) for input amino acid sequences quality check (QC)
+- [SeqKit](#seqkit) for preprocessing input amino acid sequences
```

Comment:
I do think you should mention a little bit what kind of preprocessing this involves